### PR TITLE
chore(forms): udpate files form to use elgg_view_input()

### PR DESCRIPTION
**Pull request by hypeJunction**

### CODE REVIEW

**Comment by hypeJunction** [comment]
This slightly changes the look by moving the upload limit info to the help text.
file-form

File: mod/file/views/default/forms/file/upload.php [hunk 1]
```diff
@@ -33,54 +33,67 @@
...
+	[
+		'type' => 'access',
 		'name' => 'access_id',
 		'value' => $access_id,
 		'entity' => get_entity($guid),
 		'entity_type' => 'object',
 		'entity_subtype' => 'file',
-	)); ?>
-</div>
-<div class="elgg-foot">
```
Comment (mrclay) [inline mod/file/views/default/forms/file/upload.php]: How hard would it be to keep this div?

Comment (hypeJunction) [inline mod/file/views/default/forms/file/upload.php]: It's there, see `field_class`

Comment (hypeJunction) [inline mod/file/views/default/forms/file/upload.php]: In general, I think we should figure out a way to not include the submit button within a form view. Would be some much easier to extend forms. IMO, we could pass the submit label with form form vars.

**Comment by mrclay** [comment]
LGTM

**Pull request merged**